packages/libraries/core/src/components/breadcrumb/breadcrumb-item/ods-breadcrumb-item-attributes.ts

```diff
@@ -0,0 +1,19 @@
+import { OdsComponentAttributes } from '../../ods-component-attributes';
+import { OdsIconName } from '../../icon/ods-icon-size';
+
+export interface OdsBreadcrumbItemAttributes extends OdsComponentAttributes {
+  /** contrasted or not: see component principles */
+  contrasted?: boolean,
+  /** Item link to redirect to */
+  href: string;
+  /** Icon to display */
+  icon?: OdsIconName;
+  /** @internal */
+  isCollapsed: boolean;
+  /** @internal */
+  isCollapsedItem: boolean;
```

Comment:
what is the difference with isCollapsed ?

Reply:
`isCollapsed` means that the item should not be displayed (more than 4 items), except if it is displayed as the `...` button
`isCollapsedItem` means this item is actually the `...` button

Reply:
maybe we can name `isCollapsed` as `hidden` no ?

Reply:
or isExpandableItem